packages/swayze/lib/src/core/controller/selection/user_selections/model.dart

```diff
@@ -13,6 +15,7 @@ const _uuid = Uuid();
 /// Defines a [Selection] that is controllable by a [UserSelectionState].
 abstract class UserSelectionModel extends Selection {
   /// Unique identifier of a selection in a [UserSelectionState]
+  @Deprecated('')
```

Comment:
Can we add a message? E.g., "`id` is deprecated and not used. It will be removed in the next major version"

Reply:
I had a message like this, but leaving it blank already shows a similar default message.